project/AppDependencies.scala

```diff
@@ -1,31 +1,26 @@
-import play.core.PlayVersion
-import play.sbt.PlayImport._
-import sbt.Keys.libraryDependencies
 import sbt._
 
 object AppDependencies {
 
-  private val bootstrapVersion = "7.22.0"
-  private val mongoVersion     = "1.3.0"
+  private val bootstrapVersion = "8.6.0"
+  private val mongoVersion     = "1.9.0"
 
-  val compile = Seq(
-    "uk.gov.hmrc"       %% "bootstrap-backend-play-28" % bootstrapVersion,
-    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28"        % mongoVersion,
-    "uk.gov.hmrc"       %% "domain"                    % "8.3.0-play-28"
+  val compile: Seq[ModuleID] = Seq(
+    "uk.gov.hmrc"       %% "bootstrap-backend-play-30" % bootstrapVersion,
+    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-30"        % mongoVersion,
+    "uk.gov.hmrc"       %% "domain-play-30"                    % "9.0.0"
   )
 
-  val test = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-test-play-28"     % bootstrapVersion,
-    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-test-play-28"    % mongoVersion,
-    "org.mockito"             %% "mockito-scala"              % "1.16.42",
-    "com.typesafe.play"       %% "play-test"                  % PlayVersion.current,
-    "org.scalatestplus"       %% "scalatestplus-scalacheck"   % "3.1.0.0-RC2",
-    "wolfendale"              %% "scalacheck-gen-regexp"      % "0.1.2",
-    "org.jsoup"                % "jsoup"                      % "1.12.1",
-    "org.scalatest"           %% "scalatest"                  % "3.1.4",
-    "org.scalatestplus.play"  %% "scalatestplus-play"         % "5.1.0",
-    "com.github.tomakehurst"   % "wiremock-standalone"        % "2.27.0",
-    "com.vladsch.flexmark"     % "flexmark-all"               % "0.36.8"
-  ).map(_ % "test, it")
+  def test: Seq[ModuleID] = Seq(
+    "uk.gov.hmrc"             %% "bootstrap-test-play-30"     % bootstrapVersion % Test,
+    "org.mockito"             %% "mockito-scala"              % "1.17.31" % Test,
+    "org.playframework"       %% "play-test"                  % "3.0.3" % Test,
+    "org.scalatestplus"       %% "scalatestplus-scalacheck"   % "3.1.0.0-RC2" % Test,
+    "wolfendale"              %% "scalacheck-gen-regexp"      % "0.1.2" % Test,
+    "org.jsoup"                % "jsoup"                      % "1.17.2" % Test,
+    "org.scalatest"           %% "scalatest"                  % "3.2.18" % Test,
```

Comment:
@wp94 Scalatest and "flexmark-all" can be removed, it will take it from bootstrap